content/release-10-6-0/platform-devices-10-6-0-bundle/10-6-0-21-23.md

```diff
@@ -48,7 +48,7 @@ Issue</th>
 <tr>
 <td>
 Administration</td>
-<td > In the <b>Inventory</b> tab, a user without relevant permission will no longer see the inventory tree. Moreover, the error message being displayed if a user doesn´t have relevant permissions now shows the correct information. 
In the <b>Application access</b> tab, a user without relevant permission will no longer be able to edit.</td>
+<td > In the <b>Inventory</b> tab, a user without relevant permission will no longer see the inventory tree. Moreover, the error message being displayed if a user doesn´t have relevant permissions now shows the correct information.
In the <b>Application access</b> tab, a user without relevant permission will no longer be able to edit.</td>

```

Comment:
there's an unusual character sequence in this line. looks like there should only be a space character?